packages/ui-kit/src/components/Counter/Counter.stories.tsx

```diff
@@ -2,14 +2,19 @@ import { css } from '@emotion/css'
 import type { Meta, StoryObj } from '@storybook/react'
 import React from 'react'
 import axiosInstance from '../../../../../app/storybook/src/axios'
-import { RelativeTimeRange, useStorybookAccessToken } from '../../helpers'
-import { Counter, CounterComponent } from './Counter'
+import { quotedStringRegex, RelativeTimeRange, storybookCodeTemplate, useStorybookAccessToken } from '../../helpers'
+import { Counter as CounterSource, CounterComponent } from './Counter'
 
 const meta: Meta<typeof CounterComponent> = {
   title: 'Components/Counter',
   component: CounterComponent,
-  render: (args) => <ConnectedCounterTemplate {...args} />,
-  tags: ['tag']
+  tags: ['tag'],
+  parameters: {
+    controls: { sort: 'alpha' },
+    imports: 'Counter, RelativeTimeRange',
```

Comment:
A list of imports to show in the custom context by default in all stories in the file.